src/test/java/com/wikia/webdriver/testcases/mediatests/addvideo/VetArticlePlaceholderTests.java

```diff
@@ -66,6 +67,7 @@ public void userCanAddVideoByUrlUsingPlaceholderInEditMode() {
 
   @Test(groups = {"VideoArticlePlacehoder_004"})
   @Execute(asUser = User.SUS_STAFF, onWikia = "mobileregressiontesting")
+  @RelatedIssue(issueID = "XW-4584")
```

Comment:
I think this should be fixed on prod already, could you check if test passes and remove this annotation?